yarn.lock

```diff
@@ -3559,9 +3559,12 @@ __metadata:
     dotenv: ^16.0.3
     ejs: ^3.1.8
     express: ^4.18.2
+    fs-extra: ^11.1.0
```

Comment:
I would rather avoid adding another dependency. Why is it needed? There are already some fs helpers in the Concept methods, so we could maybe extract them to utility functions and use them also in Make command?